app/components/articles/link_component.rb

```diff
@@ -7,9 +7,13 @@ def render?
     end
 
     def icon
-      return unless link
+      return unless link && %w[ebook-pdf pdflink pdf].include?(link.type)
 
-      image_tag(image_url('pdf-icon.svg'), height: '20px', alt: 'PDF') unless link.type == 'other'
+      pdf_icon
+    end
+
+    def pdf_icon
+      sanitize "<i class='bi bi-filetype-pdf link-icon' aria-label='PDF'></i>"
```

Comment:
Instead of sanitizing the HTML after the fact, could we build safe HTML? E.g. something like:

```suggestion
      tag.i class: 'bi bi-filetype-pdf link-icon', aria: { label: 'PDF' }
```

Reply:
Changed per suggestion, thanks